glow_test.go

```diff
@@ -61,3 +65,37 @@ func TestGlowFlags(t *testing.T) {
 		}
 	}
 }
+
+func TestPagerCommandValues(t *testing.T) {
+	tt := []struct {
+		pagerEnvVar     string
+		expectedCommand []string
+	}{
+		{
+			pagerEnvVar:     "C:\\Program Files\\Git\\usr\\bin\\less.exe",
+			expectedCommand: []string{"C:\\Program Files\\Git\\usr\\bin\\less.exe"},
+		},
+		{
+			pagerEnvVar:     "usr/local/bin",
+			expectedCommand: []string{"usr/local/bin"},
+		},
+		{
+			pagerEnvVar:     "",
+			expectedCommand: []string{"less", "-r"},
+		},
+		{
+			pagerEnvVar:     "",
+			expectedCommand: []string{"less", "-r"},
```

Comment:
Unless I'm missing something, the last two cases seem to be dupes.

Reply:
Oop, yes. I had a previous iteration that would take in the command line option as well, and had added it for covering that. Removed, thanks!